modules/osd-aws-privatelink-about.adoc

```diff
@@ -0,0 +1,7 @@
+[id="osd-aws-privatelink-about.adoc_{context}"]
+= Understanding PrivateLink
+
+A {product-title} cluster can be created without any requirements on public subnets or public internet access. In this configuration, Red Hat uses AWS PrivateLink (PrivateLink) to manage and monitor a cluster in order to avoid any public ingress or egress network traffic.
```

Comment:
```suggestion
A {product-title} cluster can be created without any requirements on public subnets, Internet Gateways, or NAT Gateways. In this configuration, Red Hat uses AWS PrivateLink (PrivateLink) to manage and monitor a cluster in order to avoid all public ingress network traffic.
```

@kmccarron-rh WDYT?

Reply:
Looks good - I will make a couple of minor non-content affecting changes (style).